examples/01_plotting/plot_surf_stat_map.py

```diff
@@ -120,7 +120,8 @@
     stat_map[i] = stats.pearsonr(seed_timeseries, timeseries[i])[0]
 
 # Re-mask previously masked nodes (medial wall)
-stat_map[np.where(np.mean(timeseries, axis=1) == 0)] = 0
+medial_wall_vertices = np.where(np.mean(timeseries, axis=1) == 0)
+stat_map[medial_wall_vertices] = 0
```

Comment:
unrelated: just to make it easier to read

Reply:
Do we need the np.where here ?

Reply:
not sure: it's always been there

see stable doc
https://nilearn.github.io/stable/auto_examples/01_plotting/plot_surf_stat_map.html#extracting-the-seed-time-series

Reply:
tested locally: not needed apparently